### ISSUE TITLE
Simpler trained model for inference?

### ISSUE THREAD

**Issue by stygian2a** [AdalbertoCq/Histomorphological-Phenotype-Learning#3]
Hi!
Thank you for your wonderful work!

I'm trying to load your trained networks, but I'm having some difficulties.
I managed to load the graph correctly, but I can't seem to find which tensors correspond to input/outputs... There are some gradients and Adam tensors and operations everywhere.

Would you think it would be possible for you to propose simpler weights/models if we are only looking to use it for inference?
Or, alternatively, a small script/notebook that shows how to load and use the model (Not inside your particular pipeline, but on its own.)

Thank you!

**Comment by AdalbertoCq** [comment]
Hey @stygian2a,

Thanks for the feedback :) 

If you just want to use the self-supervised pre-trained model for inference, it should be relatively easy to run [step 3 - Tile vector representations](https://github.com/AdalbertoCq/Histomorphological-Phenotype-Learning/blob/master/README_additional_cohort.md) of the flow to work with external cohorts. 

That should give you vector representations for each tile image of your own WSIs. 

Thanks,
Adal
